src/modules/Hosts/Hosts/Views/MainPage.xaml

```diff
@@ -432,13 +416,10 @@
                 <models:Entry />
             </ContentDialog.DataContext>
             <ScrollViewer
-                HorizontalScrollBarVisibility="Auto"
-                HorizontalScrollMode="Auto">
+                Padding="0,0,16,0">
                 <StackPanel
-                    Width="480"
-                    Margin="0,12,0,0"
                     HorizontalAlignment="Stretch"
-                    Spacing="24">
+                    Spacing="12">
```

Comment:
Great catch..

```suggestion
                    Spacing="16">
```

Needs to be `16` - design spec:

<img width="649" alt="image" src="https://github.com/microsoft/PowerToys/assets/9866362/734ba68e-4fe7-4c6a-8acf-07c769f285f8">

Reply:
Okay.
In _practice_, the space between the bottom of a textbox and top of the following text is already minimum 17px now.
![image](https://github.com/microsoft/PowerToys/assets/65828559/25499a4c-4c62-4e54-9f22-416473c756c6)
Looking at https://learn.microsoft.com/nl-nl/windows/apps/design/basics/content-basics, 12 seems to be more common?